pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/AbortedTxnProcessor.java

```diff
@@ -65,10 +66,9 @@ public interface AbortedTxnProcessor {
     CompletableFuture<Void> takeAbortedTxnsSnapshot(PositionImpl maxReadPosition);
 
     /**
-     * Get the lastSnapshotTimestamps.
-     * @return the lastSnapshotTimestamps.
+     * Get the snapshot stats form the processor.
      */
-    long getLastSnapshotTimestamps();
+    void getSnapshotStats(TransactionBufferStats stats);
```

Comment:
The implementation of this method is to set the property of stat. Why does it named `getxxx`?